parser/pt_filename_parser.go

```diff
@@ -0,0 +1,32 @@
+// Parse PT filename like 20170320T23:53:10Z-98.162.212.214-53849-64.86.132.75-42677.paris
+package parser
+
+import (
+	"strings"
+)
+
+type FileName struct {
+	name string
+}
+
+// GetLocalIP parse the filename and return IP.
```

Comment:
I think this would be much cleaner with a regexp parser.  But as this is already working, add a TODO to do that later.